tests/Admin/AdminTest.php

```diff
@@ -1939,7 +1939,15 @@ public function testGetFilterParametersWithoutRequest(): void
 
     public function testGetFilterFieldDescription(): void
     {
+        $labelTranslatorStrategy = $this->createStub(LabelTranslatorStrategyInterface::class);
+        $labelTranslatorStrategy
+            ->method('getLabel')
+            ->willReturnCallback(static function (string $label, string $context = '', string $type = ''): string {
+                return sprintf('%s.%s_%s', $context, $type, $label);
+            });
+
         $modelAdmin = new ModelAdmin('sonata.post.admin.model', 'Application\Sonata\FooBundle\Entity\Model', 'Sonata\FooBundle\Controller\ModelAdminController');
+        $modelAdmin->setLabelTranslatorStrategy($labelTranslatorStrategy);
```

Comment:
```suggestion
        $modelAdmin->setLabelTranslatorStrategy(new NoopLabelTranslatorStrategy());
```
Since we are not using the callback, we can use `NoopLabelTranslatorStrategy` for simplicity (it needs to import the class).

Reply:
Done